### DISCUSSION TITLE
NetBox Live Coding - Friday, August 27th

### DISCUSSION THREAD

**Discussion by jeremystretch** [netbox-community/netbox#7017] [Announcements]
Hi folks! I'll be holding another NetBox live coding session this Friday, August 27th, at 2pm UTC (10am EDT). This is a great opportunity to get a close up look at what it's like to maintain NetBox day-to-day. I also take on-topic questions live from the chat. Feel free to pop in if you'd like to learn more about NetBox behind the scenes!

https://youtu.be/BD9I3KFXZQM

(Also check out the [previous sessions](https://www.youtube.com/channel/UCyTz3NglyCnIAhvEgat41sQ/videos).)

**Comment by ziggekatten** [comment]
This is something I look forward to!
